bot/exts/evergreen/wonder_twins.py

```diff
@@ -0,0 +1,49 @@
+import random
+from pathlib import Path
+
+import yaml
+from discord.ext.commands import Bot, Cog, Context, command
+
+
+class WonderTwins(Cog):
+    """Cog for a Wonder Twins inspired command."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+        with open(Path.cwd() / "bot" / "resources" / "evergreen" / "wonder_twins.yaml", "r", encoding="utf-8") as f:
+            info = yaml.load(f, Loader=yaml.FullLoader)
+            self.water_types, self.objects, self.adjectives = info["water_types"], info["objects"], info["adjectives"]
```

Comment:
```suggestion
            self.water_types, self.objects, self.adjectives = info
```
This looks clearer to me. If you don't like the proposed solution, I think putting it on 3 lines would look the best. The original code had a bit too much going on in that single line, in my opinion.